gnovm/pkg/gnomod/pkg.go

```diff
@@ -6,13 +6,16 @@ import (
 	"os"
 	"path/filepath"
 	"strings"
+
+	"github.com/gnolang/gno/gnovm/pkg/gnoimports"
+	"github.com/gnolang/gno/gnovm/pkg/gnolang"
 )
 
 type Pkg struct {
-	Dir      string   // absolute path to package dir
-	Name     string   // package name
-	Requires []string // dependencies
-	Draft    bool     // whether the package is a draft
+	Dir     string // absolute path to package dir
+	Name    string // package name
+	Imports []string
```

Comment:
I'm a bit confused how this wasn't already part of `Pkg`, but alright :)

Reply:
`Require` was enough I guess